assets/js/back2top.js

```diff
@@ -0,0 +1,13 @@
+var btn = $('#backToTop');
```

Comment:
Let's avoid using a global variable here. 

@RandilaP you can explain why.

Reply:
So @jayasanka-sack we can use there a 'let' without the 'var' right?